packages/teams-js/src/public/pages.ts

```diff
@@ -640,7 +642,12 @@ export namespace pages {
 
     function handleBackButtonPress(): void {
       if (!backButtonPressHandler || !backButtonPressHandler()) {
-        navigateBack();
+        if (Communication.childWindow) {
+          // If no parent window function handled it let the child window
```

Comment:
I think this should be current window, right? I would interpret "Parent" as "parent of the current window", which I don't think is what you mean

Reply:
Yes, I will correct that comment.  I was thinking pare of the child window that I am sending it too.